### DISCUSSION TITLE
Exploring Network Slicing with Srsran , Seeking Guidance and Assistance

### DISCUSSION THREAD

**Discussion by rajugbv** [srsran/srsRAN_Project#358] [General Help] [answered]
Hi everyone,

I successfully implemented the srsran project using two USRP 2901 devices. Now, I want to implement a small slicing feature. I want to know if I can implement network slicing using the Above project. I didn't find any useful tutorials, so can someone please help me with this?

Thank you.

**Comment by ismagom** [comment, marked as answer]
There is currently no slicing documentation.
